### PR TITLE
Release notes fixup

### PR DESCRIPTION
**Pull request by egernst**

### CODE REVIEW

File: release/runtime-release-notes.sh [hunk 1]
```diff
@@ -83,15 +83,30 @@ print_release_notes() {
...
+	cat <<EOT
+
 ## Compatibility with Docker
 Kata Containers ${runtime_version} is compatible with Docker ${docker_version}
 
 ## Compatibility with CRI-O
 Kata Containers ${runtime_version} is compatible with CRI-O ${crio_version}
 
-## Compatibility with cri-contaienrd
+## Compatibility with cri-containerd
```
Comment (jodh-intel) [inline release/runtime-release-notes.sh]: Good catch!

**Comment by jodh-intel** [comment]
lgtm

**Comment by jodh-intel** [comment]
Hi @egernst - I notice that this PR includes #173 - do you and @jcvenegas want to fight it out? :smile:

**Comment by jodh-intel** [comment]
:bell: :bell:

**Comment by grahamwhaley** [comment]
big green button there is....

**Comment by jodh-intel** [comment]
A nugget of purest green...!

**Pull request merged**